projects/storefrontlib/src/cms-components/content/link/link.component.html

```diff
@@ -1,6 +1,6 @@
-<a
+<cx-generic-link
```

Comment:
We could also add `rel="noopener"` for external links using `target="_blank"`

Reply:
updated in generic-link.component.html